opencti-platform/opencti-front/src/private/components/threats/threat_actors_individual/ThreatActorIndividual.tsx

```diff
@@ -155,61 +156,98 @@ const ThreatActorIndividualComponent = ({
+                      threatActorIndividualData={threatActorIndividual}
+                    />
+                  </Grid>
+                );
+              case 'basicInformation':
+                return (
+                  <Grid item={true} xs={6} style={{ paddingTop: 10 }}>
+                    <StixDomainObjectOverview stixDomainObject={threatActorIndividual} />
+                  </Grid>
+                );
+              case 'demographics-biographics':
+                return (
+                  <>
+                    { hasDemographicsOrBiographics(threatActorIndividual) && (
+                      <>
+                        <Grid item={true} xs={width} style={{ marginTop: 30 }}>
+                          <ThreatActorIndividualDemographics
+                            threatActorIndividual={threatActorIndividual}
+                          />
+                        </Grid>
+                        <Grid item={true} xs={width} style={{ marginTop: 30 }}>
+                          <ThreatActorIndividualBiographics
+                            threatActorIndividual={threatActorIndividual}
+                          />
+                        </Grid>
+                      </>
+                    ) }
+                  </>
+                );
+              case 'latestCreatedRelationships':
+                return (
+                  <Grid item={true} xs={width} style={{ marginTop: 30 }}>
+                    <SimpleStixObjectOrStixRelationshipStixCoreRelationships
+                      stixObjectOrStixRelationshipId={threatActorIndividual.id}
+                      stixObjectOrStixRelationshipLink={`/dashboard/threats/threat_actors_individual/${threatActorIndividual.id}/knowledge`}
+                    />
+                  </Grid>
+                );
+              case 'latestContainers':
+                return (
+                  <Grid item={true} xs={width} style={{ marginTop: 30 }}>
+                    <StixCoreObjectOrStixRelationshipLastContainers
+                      stixCoreObjectOrStixRelationshipId={threatActorIndividual.id}
+                    />
+                  </Grid>
+                );
+              case 'externalReferences':
+                return (
+                  <Grid item={true} xs={width} style={{ marginTop: 30 }}>
+                    <StixCoreObjectExternalReferences
+                      stixCoreObjectId={threatActorIndividual.id}
+                    />
+                  </Grid>
+                );
+              case 'mostRecentHistory':
+                return (
+                  <Grid item={true} xs={width} style={{ marginTop: 30 }}>
+                    <StixCoreObjectLatestHistory
+                      stixCoreObjectId={threatActorIndividual.id}
+                    />
+                  </Grid>
+                );
+              case 'notes':
+                return (
+                  <Grid item={true} xs={width} style={{ marginTop: -20 }}>
```

Comment:
I don't understand all those different styles

Reply:
The Notes component was not originally part of the Grid children.
When including it, the style broke.

Reply:
Plus, different style are set depending on the child component.
![image](https://github.com/user-attachments/assets/54eac991-9817-42cd-8f93-286cf92ae35b)

Reply:
I will investigate into why the style was managed this way.

Reply:
The investigation can be found in the brainstorm.
The code will be rebased on the Grid style refactor branch.